assets/scss/lib/partials/_search.scss

```diff
@@ -56,3 +56,56 @@
     display: none;
   }
 }
+
+// Search page form.
+#views-exposed-form-localgov-sitewide-search-sitewide-search-page {
+  max-width: 48em;
+  padding: 1em 2em 2em 2em;
+
+  .form-actions {
+    padding-left: 0;
+  }
+
+  @include media-breakpoint-up(md) {
+    .form-text {
+      float: left;
+      width: 80%;
+      height: 3em;
+      margin: 0;
+      padding: 0.5em 1em;
+    }
+
+    .form-actions {
+      clear: none;
+      float: right;
+      height: 3em;
+      width: 20%;
+      margin: 0;
+
+      .button {
+        margin-top: 0 !important;
+        padding-top: 0.45em;
+        padding-bottom: 0.4em;
+
+        &:after {
+          top: 0.5em;
+        }
+      }
+    }
+  }
+}
+
+// Search results.
+ol.search-results {
+  padding: 0;
+  list-style-type: none;
+
+  .node--view-mode-search-result {
```

Comment:
It would be good if this template had an item in the `classes` array for this node type called `search-result`, then we could use that as our `Block` class in BEM.

Reply:
Yep - using BEM classes now.